<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CIOCfgForm</class>
 <widget class="QWidget" name="CIOCfgForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Dodatkowe wejścia</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>350</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>275</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_62">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>35</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wejście:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="dorIOMod">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>30</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="dorChk">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>230</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kontrola otwarcia drzwi szafy</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="dorBitNo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_63">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nr modułu IO:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>275</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_64">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>35</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wejście:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="breakInIOMod">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>30</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="breakInChk">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>240</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kontrola otwarcia skrzyni trafo</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="breakInBitNo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_65">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nr modułu IO:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_9">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>275</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_70">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>35</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wejście:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="handCtrlIOMod">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>30</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="handCtrlChk">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sterowanie ręczne</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="handCtrlBitNo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_71">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nr modułu IO:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_10">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>275</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_72">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>35</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wejście:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="localCtrlIOMod">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>30</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="localCtrlChk">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>strowanie lokalne</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="localCtrlBitNo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_73">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nr modułu IO:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>275</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_68">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>35</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wejście:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="remoteCtrlIOMod">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>30</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="remoteCtrlChk">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sterowanie zdalne</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="remoteCtrlBitNo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_69">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nr modułu IO:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>275</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_66">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>35</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wejście:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="surgeProtectIOMod">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>30</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="surgeProtectChk">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>260</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kontrola zab. antyprzepięciowego</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="surgeProtectBitNo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_67">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nr modułu IO:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_11">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>275</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_74">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>35</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wejście:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="lockCtrlIOMod">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>30</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="lockCtrlChk">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Blokada sterowania</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="lockCtrlBitNo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_75">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nr modułu IO:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
